frontend/ui/impl/src/test/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorReportSubmitterTest.java

```diff
@@ -0,0 +1,42 @@
+package com.virtuslab.gitmachete.frontend.ui.impl;
+
+import static org.mockito.ArgumentMatchers.*;
+
+import java.util.HashMap;
+
+import com.intellij.diagnostic.AbstractMessage;
+import com.intellij.diagnostic.IdeaReportingEvent;
+import com.intellij.diagnostic.LogMessage;
+import com.intellij.openapi.diagnostic.IdeaLoggingEvent;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({GitMacheteErrorReportSubmitter.class})
+public class GitMacheteErrorReportSubmitterTest {
+  @Test
+  public void testGitHubIssueUriConstruction() throws Exception {
+    IdeaLoggingEvent[] events = new IdeaLoggingEvent[1];
+
+    AbstractMessage messageObject = Whitebox.newInstance(LogMessage.class);
+
+    events[0] = new IdeaReportingEvent(messageObject, /* message */ null, /* stacktrace */"", /* plugin */null);
+
+    GitMacheteErrorReportSubmitter reportSubmitter = PowerMockito.spy(new GitMacheteErrorReportSubmitter());
+
+    PowerMockito
+        .stub(PowerMockito.method(GitMacheteErrorReportSubmitter.class, "getTemplateVariables", IdeaLoggingEvent[].class,
+            String.class))
+        .toReturn(new HashMap<String, String>());
+
+    String reportBody = Whitebox.invokeMethod(reportSubmitter, "getReportBody", events, /* additionalInfo */ null);
```

Comment:
Hmmm back to the original issue #972... will this test detect a typo in resource name?

Reply:
Now it will detect a typo in resource's name.